modules/preview-react/radio/lib/StyledRadioButton.tsx

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import {
+  styled,
+  StyledType,
+  focusRing,
+  mouseFocusBehavior,
+  createComponent,
+  ExtractProps,
+  Themeable,
+} from '@workday/canvas-kit-react/common';
+import {colors, inputColors, spaceNumbers, borderRadius} from '@workday/canvas-kit-react/tokens';
+
+import {Box, Flex} from '@workday/canvas-kit-react/layout';
+
+const radioWidth = 18;
+const rippleRadius = (spaceNumbers.l - radioWidth) / 2;
+const radioHeight = 18;
+
+export interface StyledRadioButtonProps extends ExtractProps<typeof Box, 'input'>, Themeable {
+  variant?: 'inverse' | undefined;
+}
+
+export const StyledRadioInput = styled(Box.as('input'))<StyledRadioButtonProps & StyledType>(
+  {
+    '&:focus, &:active': {
+      outline: 'transparent',
+    },
+  },
+  ({
+    disabled,
+    variant,
+    theme: {
+      canvas: {
+        palette: {
+          primary: themePrimary,
+          common: {focusOutline: themeFocusOutline},
+        },
+      },
+    },
+  }) => ({
+    cursor: disabled ? undefined : 'pointer',
```

Comment:
Maybe add `undefined` value to the end, like this: `!disabled ? 'pointer' : undefined`.

Q: is it behave differently if you do `disabled && 'pointer'`?

Reply:
I'm curious if Emotion cleans `false` values

Reply:
ill give it a try!

Reply:
So TIL it doesn't clean false value, it gives a type error saying that `false` is not assignable to `cursor`

Reply:
I went with `!disabled ? 'pointer' : undefined` it reads better!

Reply:
Sounds good! 👍